testsuite/tests/kuadrant/authorino/operator/tls/conftest.py

```diff
@@ -117,12 +144,42 @@ def custom_cert(certificates):
     return certificates["custom_cert"]
 
 
+@pytest.fixture(scope="module")
+def intermediate_valid_cert(certificates):
```

Comment:
No need to create fixture if you need them only once (not sure if this is the case)